controls/anssi.yml

```diff
@@ -845,7 +845,8 @@ controls:
   - id: R63
     level: intermediary
     title: Explicit arguments in sudo specifications
-    # rules: TBD
+    rules:
+    - sudoers_no_command_args
```

Comment:
How about changing rule id to `sudoers_explicit_command_args`, per rule title?

Reply:
I agree, that would be a more descriptive ID.